tests/phpunit/tests/theme/base.php

```diff
@@ -45,6 +45,7 @@ public function tear_down() {
 
 		wp_clean_themes_cache();
 		wp_clean_theme_json_cache();
+		wp_clean_theme_json_cache();
```

Comment:
This is now redundant and can be removed.

Reply:
Fixed now.